src/Microsoft.CodeAnalysis.NetAnalyzers/src/Microsoft.CodeAnalysis.NetAnalyzers/Microsoft.NetCore.Analyzers/xlf/MicrosoftNetCoreAnalyzersResources.de.xlf

```diff
@@ -2238,6 +2238,26 @@ Erweiterungen und benutzerdefinierte Konvertierungen werden bei generischen Type
         <target state="translated">Ziehen Sie "IsEmpty" gegenüber "Count" vor</target>
         <note />
       </trans-unit>
+      <trans-unit id="PreferJsonElementParseDescription">
+        <source>JsonDocument implements IDisposable and needs to be properly disposed. When only the RootElement is needed, prefer JsonElement.Parse which doesn't require disposal.</source>
```

Comment:
Not super familiar with how this repo works, but is putting English here the appropriate approach? I presume it will get picked up and translated by the relevant team?

Reply:
These are all auto-generated by the build tooling. These aren't intended to be manually authored or edited by us.